### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by RW2023**
Edit Readme and add contribution instructions. Also added a screenshot

### CODE REVIEW

**Comment by Yourmix0x** [comment]
Updated README by @RW2023

File: toDo.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+2. Implement an onChange event listener on the input field to update the value of a separate state variable to track the input field's value.
+3. Implement an onSubmit event listener on the form that adds the new task to the task list state variable and clears the input field's value.
+4. Update the rendering of the task list to display the dynamically generated tasks from the state variable.
+
+### Implement functionality for editing and deleting tasks:
+
+1. Create a state variable in the Form component to store the list of tasks.
+2. Implement an onClick event listener on the "Edit" button that toggles the editable state of the corresponding task.
+3. Implement an onChange event listener on the corresponding task's input field to update the value of the task.
+4. Implement an onClick event listener on the "Delete" button that removes the corresponding task from the task list state variable.
```
Comment (YourmixJNR) [inline toDo.md:17]: Linethrough the task to mark as completed and vanish after some sec...
What's your view?

Comment (Yourmix0x) [inline toDo.md:17]: Linethrough the task to mark as completed and vanish after some sec...
What's your view?

Comment (RW2023) [reply toDo.md:17]: That sounds perfect 😀 I am still a react rookie so I can't help much with implementation. Yet

**Comment by YourmixJNR** [comment]
Update README

**Pull request closed without merging**